AsyncSwift/Observed/AppData.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  AppData.swift
+//  AsyncSwift
+//
+//  Created by Inho Choi on 2022/09/08.
+//
+
+import Foundation
+import UIKit
+
+class AppData: ObservableObject {
+	@Published var currentTab: Tab = .event
+	
+}
+
+
+enum Tab: String {
+	case event = "event"
+	case ticketing = "ticketing"
+	case stamp = "stamp"
```

Comment:
```suggestion
	case event
	case ticketing
	case stamp
```

같은 값의 경우에는 생략 가능해요 :  )